statics/js/components/topology.js

```diff
@@ -285,6 +285,21 @@ var TopologyComponent = {
 
     this.setGremlinFavoritesFromConfig();
 
+    if (typeof(this.$route.query.highlight) !== "undefined") {
+      this.topologyEmphasize = this.$route.query.highlight;
+      setTimeout(function(){self.emphasizeGremlinExpr();}, 2000)
```

Comment:
I don't think the timeout is needed as WebSocket event will trigger the function ? By the way I didn't have it working, It triggers exceptions, please check the console to see if it the same for you

Reply:
- Hi Sylvain, when I wrote the original code I didn't had the timeout in place, and was thinking (like you) that there is no need, later I discovered that it just doesn't work with time-out., I checked 1 second and it worked, but decide on 2 seconds just to be on the safe side (also, it looks visually nice to see the nodes, and than the highlight appears).

- With regards to the exception, can you share ... I am running a version that is 99% similar to this one (only change is in the PathPrefix("/topology") that should make a difference) and I don't get any exception ....

Reply:
I really think the setTimeout is useless but that's fine to let it I guess, here the exception I get, without or with 

```
topology.js:452 Uncaught TypeError: Cannot read property 'indexOf' of null
    at a.endsWith (topology.js:452)
    at a.n [as endsWith] (vue.min-2.1.9.js:6)
    at a.emphasizeGremlinExpr (topology.js:595)
    at a.n [as emphasizeGremlinExpr] (vue.min-2.1.9.js:6)
    at later (utils.js:8)
```

Reply:
The issue is not due to your patch, can you fix it with this change the line https://github.com/skydive-project/skydive/blob/master/statics/js/components/topology.js#L463

by this one 
```
if (favorite) self.topologyEmphasize = favorite;
```

BTW a ";" is missing here at the end of the setTimeout line

Reply:
But we already protect against `favorite` being undefined here: https://github.com/skydive-project/skydive/blob/master/statics/js/components/topology.js#L455

Plus such a fix (if needed) this outside the context of the current PR. (so can be addressed separately).

Reply:
Ok I will do